doc/advanced.rst

```diff
@@ -136,6 +136,10 @@ Creating a filter is as simple as associating a name with a PHP callable::
     // or a simple PHP function
     $filter = new Twig_SimpleFilter('rot13', 'str_rot13');
 
+    // or a class static method
+    $filter = new Twig_SimpleFilter('rot13', array('SomeClass', 'rot13Filter'));
```

Comment:
It looks you only need the static example one below. The above one is already added on 143. Or did I miss something else ?